docs/modules/vault.md

```diff
@@ -0,0 +1,33 @@
+# Vault Module
+
+[Vault](https://www.vaultproject.io/) by HashiCorp is a tool for securely accessing secrets such as API keys, passwords, or certificates. This module allows you to run and initialize a Vault container for integration tests.
+
+## Install
+
+```bash
+npm install @testcontainers/vault --save-dev
+```
+
+## Examples
+
+<!--codeinclude-->
+
```

Comment:
I wonder if these empty lines are the reason there is whitespace between each line of code: https://deploy-preview-1079--testcontainers-node.netlify.app/modules/vault/. Could we try removing them?

Reply:
Yes, that was the cause. Great catch!

https://github.com/testcontainers/testcontainers-node/pull/1079/commits/85c8091f8e68ffbab5459a376f357b3a1d92987b